Google.Api.Gax.Grpc.Tests/CallSettingsTest.cs

```diff
@@ -273,7 +274,7 @@ internal static void AssertSingleHeader(CallSettings callSettings, string expect
             Assert.Equal(expectedHeaderValue, metadata[0].Value);
         }
 
-        internal static void AssertRoutingHeader(CallSettings callSettings, string expectedHeaderValue)
+        internal static void AssertRoutingHeaderIgnoringOrder(CallSettings callSettings, string expectedHeaderValue)
```

Comment:
To be honest I'd have preferred to make the order deterministic instead, but we can go with this for now.
(I'm hoping that once most tests have moved, everything will be simpler all round.)

Reply:
👍